scripts/installer/Makefile

```diff
@@ -10,12 +10,15 @@ SOLR_SCHEMA=${INSTALLER_ZIP_DIR}/schema.xml
 SOLR_CONFIG=${INSTALLER_ZIP_DIR}/solrconfig.xml
 INSTALL_SCRIPT=${INSTALLER_ZIP_DIR}/install
 
+DATAVERSE_VERSION=4.9
```

Comment:
As I mentioned at http://irclog.iq.harvard.edu/dataverse/2018-06-08#i_68676 I'd really rather not hard code a version number in yet another place. Already at http://guides.dataverse.org/en/4.9/developers/making-releases.html#bump-version-numbers the number has to be incremented in four places and I'd really rather not add a fifth.

Somewhere I figured out how to get the build number out of the pom.xml with `xpath` but I can't find it at the moment. Sorry.

Reply:
Sounds reasonable - thanks for reviewing.